procrastinate/worker.py

```diff
@@ -365,10 +407,13 @@ async def _shutdown(self, side_tasks: list[asyncio.Task]):
 
     def _start_side_tasks(self) -> list[asyncio.Task]:
         """Start side tasks such as periodic deferrer and notification listener"""
-        side_tasks = [asyncio.create_task(self.periodic_deferrer(), name="deferrer")]
-        if self.wait and self.listen_notify:
+        side_tasks = [
+            asyncio.create_task(self.periodic_deferrer(), name="deferrer"),
```

Comment:
should `periodic_deferrer` start with a `_` too ? Is there a rationale on what gets a `_` vs on on a class that's itself internal ?

Reply:
Not sure about Python. Coming from a different background, I see `_` being used when it is not meant to be used outside the class, whether the class is internal or not. 
Changing to `_periodic_deferrer` but happy to go with whatever convention you prefer here.